pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+once Pirates and Privateers took control of the vessels. The picture shows the changes 
+did not occur until the ship reached the region and ported in a different location than 
+non-pirated ships.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/Average%20Price%20of%20Slaves%20Over%20Time.jpg)
+![image]({{ site.baseurl}}/assets/images/Pirate%20Slave%20activity%20.png)
+
+These two images compare the price of slaves between 1804-1861 and the
+overall sale of pirated ships from 1700-1827. The standard tread is that, 
+by year, there was no major significant increase to sold pirated ships when 
+comparing the drop and rise on the cost of slave worth. The image shows pirates
+were not investing considerable energy to gather wealth by the slave trade. 
+Moreover, the representation does indicate the random interaction between pirates
+and slave ships. The number of slaves captured and sold by Pirates is deficient 
+compared to the significant traffic of slave ships in the area.
 
-This is where you where you discuss any (initial) conclusions you have come to. This is also a natural place to put any and all visualizations that you come up with. In fact, this is where you tell the reader what it is you discovered. You can use the first person if that helps you discuss the process. This is where you both describe to the reader what they are looking at (if they are looking at a visualization) and, most importantly, what your interpretation of the results is.
+---
 
-If you want to add an image you can do it like this....
+![image]({{ site.baseurl}}/assets/images/Pirates%20and%20actions%20with%20slaves.png)
 
-![Example Image]({{ site.baseurl }}/assets/images/example-image.png)
+The Relative Density comparing all pirated ships and its cargo shows that there
+was not a significant influx to ships with slaves and ships without slaves. As 
+described above, the region was covered with ships that carried slave in around 
+the slave market. The lack of significant to ships with and without slaves shows 
+that pirating slaves were not the main focus of pirates and privateers.
 
-Whether you turn your visualizations into static pictures and put them here or embed them, you MUST discuss your visualizations adequately. That means that whoever is the visualization expert must explain what they think the visualization means. You should explain anything that is not self-apparent from the picture alone. Moreover, you should at least comment on whether you think you can draw broader conclusions from any of the visualizations, either when considered individually or all together.
+---
 
-If your project was less visualization-centric, this is where you at least explain in plain words what you learned through the process of non-visual analysis. For example, if you used text mining not for visualization purposes, but to help you manually find interesting threads of conversation in a body of documents.
+![image]({{ site.baseurl}}/assets/images/Tableau%20Bars%20.png)
```

Comment:
This dashboard feels a bit all over the place. The topics are all related, but somehow having a bunch of bar charts that are all on different scales and axes makes it look a bit like a geometric painting.

I'm not sure what it is... the information you have is valuable, but something about how it all comes together feels confusing to the eye. I know that isn't a very helpful criticism, but just letting you know how it reads.